app/models/agents/user_location_agent.rb

```diff
@@ -6,19 +6,17 @@ class UserLocationAgent < Agent
 
     gem_dependency_check { defined?(Haversine) }
 
-    description do
-      <<-MD
-        #{'## Include `haversine` in your Gemfile to use this Agent!' if dependencies_missing?}
-        The UserLocationAgent creates events based on WebHook POSTS that contain a `latitude` and `longitude`.  You can use the [POSTLocation](https://github.com/cantino/post_location) or [PostGPS](https://github.com/chriseidhof/PostGPS) iOS app to post your location.
+    description <<-MD
```

Comment:
This needs to be in a block (`do`...`end`) so that the interpolation below of `user.id` can work, I think.  If you view this Agent, is the `https://\#{ENV['DOMAIN']}/users/\#{user.id}/update_location/:secret` part right?